api/v1alpha1/jobset_webhook.go

```
@@ -81,7 +83,23 @@ func (js *JobSet) ValidateCreate() error {
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
 func (js *JobSet) ValidateUpdate(old runtime.Object) error {
-	return nil
+	oldObjCopy := old.DeepCopyObject().(*JobSet)
+	if pointer.BoolDeref(oldObjCopy.Spec.Suspend, false) {
+		return nil
```

we need to return an error if there was an attempt to do any update